### PR TITLE
packages/git-service: add tests

### PR DESCRIPTION
**Pull request by debsmita1**
has tests for:
- bitbucket-service.ts
- gitlab-service.ts
- github-service.ts

Refer Jira tasks:
- https://issues.redhat.com/browse/ODC-2654
- https://issues.redhat.com/browse/ODC-2671
- https://issues.redhat.com/browse/ODC-2672

### CODE REVIEW

File: frontend/packages/git-service/src/services/__tests__/__nock-fixtures__/gitlab/repo-not-reachable.json [hunk 1]
```diff
@@ -0,0 +1,50 @@
+[
```
Comment (abhinandan13jan) [inline frontend/packages/git-service/src/services/__tests__/__nock-fixtures__/gitlab/repo-not-reachable.json]: __tests__/__nock-fixtures . why this name?

**Comment by spadgett** [comment]
/hold
due to a bug in CI

**Pull request merged**